packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -115,14 +117,14 @@ const RecentCICDRunsSwitcher = ({ entity }: { entity: Entity }) => {
 };
 
 const OverviewContent = ({ entity }: { entity: Entity }) => (
-  <Grid container spacing={3}>
+  <Grid container spacing={3} alignItems="stretch">
     <Grid item md={6}>
       <AboutCard entity={entity} />
     </Grid>
     <RecentCICDRunsSwitcher entity={entity} />
     {isLighthouseAvailable(entity) && (
       <Grid item sm={4}>
-        <LastLighthouseAuditCard />
+        <LastLighthouseAuditCard variant={GRID_CARD_VARIANT} />
```

Comment:
It's unfortunate if this has to be done everywhere tbh, and plenty of cards are gonna miss it too.

My initial question here is, can we make the height100 variant the default? And going a bit further I'm wondering if we can get rid of the varian altogether, they're a bit awkward x)

Reply:
I completely agree with you. We could let decide the plugin itself, if the card is intended for use in a grid and provide a flag if the default is not desired. 

If it is the default setting in the InfoCard, I am not sure what consequences this might have on InfoCards that are not used in a grid.

Reply:
Yeah idd, would have to map out the existing usage of InfoCards.

Regarding the change in this PR, I think I'd rather have a `variant='grid-item'` instead of exporting the constant from core.